### PR TITLE
Shooter tuning stuff

### PR DESCRIPTION
**Pull request by Stubbynarwhal**

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -46,12 +46,12 @@ public class RobotContainer {
...
-  private final PrepCommand secondShoot = new PrepCommand(shooterSubsystem, 30, 0); //TODO Tune for actual angles
-  private final PrepCommand thirdShoot = new PrepCommand(shooterSubsystem, 30, 0); //TODO Tune for actual angles
-  private final PrepCommand closeShoot = new PrepCommand(shooterSubsystem, 30, 0); //TODO Tune for actual angles
-  private final PrepCommand ampShoot = new PrepCommand(shooterSubsystem, 30, 0); //TODO Tune for actual angles
-  private final PrepCommand speakerShoot = new PrepCommand(shooterSubsystem, 30, 0); //TODO Tune for actual angles
+  private final PrepCommand firstShoot = new PrepCommand(shooterSubsystem, 30, 0.3); //TODO Tune for actual angles
+  private final PrepCommand secondShoot = new PrepCommand(shooterSubsystem, 30, 0.3); //TODO Tune for actual angles
+  private final PrepCommand thirdShoot = new PrepCommand(shooterSubsystem, 30, 0.3); //TODO Tune for actual angles
+  private final PrepCommand trapShoot = new PrepCommand(shooterSubsystem, 66, 1); //TODO Tune for actual angles
+  private final PrepCommand ampShoot = new PrepCommand(shooterSubsystem, 59.6, 0.25); //TODO Tune for actual angles
```
Comment (centiwati) [inline src/main/java/frc/robot/RobotContainer.java:53]: We might want to change this... I've noticed it's not super consistent. Either slower speed or different angle, or both.

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -142,7 +144,6 @@ public void setControllerVibrations() {
    * @return the command to run in autonomous
    */
   public Command getAutonomousCommand() {
-    return null;
-//    return autoChooser.getSelected();
+    return autoChooser.getSelected();
```
Comment (centiwati) [inline src/main/java/frc/robot/RobotContainer.java:147]: We want to comment this out I believe. @awfulcarpet You should work on figuring out why this doesn't work and how we should do it instead.

File: src/main/java/frc/robot/commands/ShootCommand.java [hunk 3]
```diff
@@ -11,7 +11,7 @@
 public class ShootCommand extends Command {
   private final Shooter shooter;
   private final Intake intake;
-  private double speed = 1; // TODO Change to intake motor speed (in rotations per second)
+  private double speed = 0.2; // TODO Change to intake motor speed (in rotations per second)
```
Comment (centiwati) [inline src/main/java/frc/robot/commands/ShootCommand.java:14]: This isn't rotations per second but percentage of power.

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 4]
```diff
@@ -102,8 +105,16 @@ public void periodic() {
                     }
                 }
 
-                if (timer.hasElapsed(0.1)) { // TODO: tune this value. This timer starts when the note hits the last sensor, and this value stops the intaking process when the timer reaches this value
-                    currentState = IntakeState.IDLE;
+                if (timer.hasElapsed(0.15) && ! noteHitShooter) { // TODO: tune this value. This timer starts when the note hits the last sensor, and this value will make the indexer go backwards for a short period of time to make sure the note is in the right spot.
```
Comment (centiwati) [inline src/main/java/frc/robot/subsystems/Intake.java:108]: This timer needs to be tuned a bit. @GucciBanana11 Decrease the value a bit.

File: src/main/java/frc/robot/subsystems/Intake.java [hunk 5]
```diff
@@ -102,8 +105,16 @@ public void periodic() {
...
+                    noteHitShooter = true;
+                    intakeMotor.set(-0.1);
+                    timer.stop();
+                    timer.reset();
+                    timer.start();
+                }
+
+                if (timer.hasElapsed(0.15) && noteHitShooter) { // TODO: tune this value. This timer starts when the note hits the shooter wheels, and this value stops the intaking process when the timer reaches this value
```
Comment (centiwati) [inline src/main/java/frc/robot/subsystems/Intake.java]: Increase this time by a bit @GucciBanana11

**Review by centiwati — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/RobotContainer.java (hunk 2 above)
Comment (awfulcarpet) [reply src/main/java/frc/robot/RobotContainer.java:147]: is fixed in #42

**Comment by GucciBanana11** [comment]
To be fixed with new PR

**Pull request closed without merging**